discovery_engine/lib/src/domain/document_manager.dart

```diff
@@ -155,13 +155,25 @@ class DocumentManager {
   /// Fails if [sec] is negative or [id] does not identify an active document.
   Future<void> addActiveDocumentTime(
     DocumentId id,
-    DocumentViewMode mode,
+    DocumentViewMode viewMode,
     int sec,
```

Comment:
:art: since the `mode` is renamed, this could be renamed as well (maybe clarify in the docs that this time is meant to be in seconds)
```suggestion
    int viewTime,
```